frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -25,9 +25,8 @@ export const checkError = (data: DataState): ErrorType[] => {
 };
 
 export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): boolean => {
-  const independent: string =
-    csv.metadata.annotations?.['external.cluster.ocs.openshift.io/supported'];
-  return independent === 'true';
+  const independent: string = csv.metadata.annotations?.['features.ocs.openshift.io/enabled'];
+  return independent.includes('external');
```

Comment:
I think we should split the key list and then check. If any other flag wants to support external as a keyword then it will be problematic.

link in case of a snapshot, once feature can be snapshot another can be snapshot-scheduling. Both will be enabled with this idea. Changes of this is less I agree but still want to consider it.

@bipuladh @a2batic

Reply:
We are not setting any flag here.

Reply:
We are just checking for annotation to enable some other part of code here.

Reply:
Then if we are setting the INDEPENDENT mode flag can't we use the same in OCS install-page.tsx to see if it's true or not?

Reply:
We have two flags for Independent mode. One comes from OCS CR. This is used to activate the dashboard. 
We use the annotation to only unlock flows for Independent Mode Installation. No flag is required. We can move this code to feature.ts. But we will have to connect it to the redux store. I don't want to overcomplicate things in this PR.